src/asmcnc/keyboard/layouts/qwertyKR.json

```diff
@@ -0,0 +1,66 @@
+{
+    "title": "Qwerty",
+    "description": "A classical US Keyboard",
```

Comment:
@VladPavelescu

Reply:
The "description" isn't accurate because it is not a classical US layout, that was just copied over when I made the file and I overlooked it. However, there is a need for a qwerty layout specifically for Korean because they have an extra button called "Han/Yeong" (한/영) which basically switches the layout between Korean characters and the Latin characters. This button is usually to the right of the spacebar. There is another button that is supposed to be to the left of the spacebar which is called a "Hanja key" (한자) which converts Korean words to Chinese characters. I haven't implemented the Hanja key from the beginning because I didn't think it was necessary (however I now think that was a mistake as the Hanja key is standard on Korean keyboards).